letsdrill.rb

```diff
@@ -1,24 +1,27 @@
 def get_letter_grade(integer)
- 
-  #Put your code here!
-
+  if integer >= 90 then 'A'
+  elsif integer >= 80 then 'B'
+  elsif integer >= 70 then 'C'
+  elsif integer >= 60 then 'D'
+  elsif integer >= 50 then 'F'
+  else 'Invalid grade'
+  end
 end
 
 def shortest_string(array)
-
-  #Put your code here!
-
+  array.sort { |x,y| x.length <=> y.length }.shift
```

Comment:
This works.  Check out `sort_by`, `min`, and `min_by` here: http://ruby-doc.org/core-2.2.3/Enumerable.html.